src/installation/guides/arm-devices/index.md

```diff
@@ -27,46 +28,45 @@ thoroughly.
 ### Pre-built images
 
 After [downloading and verifying](../../index.md#downloading-installation-media)
-an image, it can be flashed onto an SD card in `/dev/mmcblk0`, for example, with
-the [cat(1)](https://man.voidlinux.org/cat.1),
+an image, it can be flashed onto an SD card located in the `/dev/` directory
```

Comment:
I like the change, because `/dev/mm...` was an example, but the SD card was also an example, since someone could be using an eMMC module or some USB drive.

Reply:
Ah, okay. Changed to:

> After [downloading and verifying](../../index.md#downloading-installation-media) an image, it can be written to the relevant media with [cat(1)](https://man.voidlinux.org/cat.1), [pv(1)](https://man.voidlinux.org/pv.1), or [dd(1)](https://man.voidlinux.org/dd.1). For example, to flash it onto an SD card located at `/dev/mmcblk0`:

Reply:
Great!